vsphere/resource_vsphere_distributed_port_group_test.go

```diff
@@ -171,6 +171,53 @@ func TestAccResourceVSphereDistributedPortGroup(t *testing.T) {
 				},
 			},
 		},
+		{
+			"single_attribute",
```

Comment:
I think we need to be more descriptive in the sub-tests here - _tag_ is descriptive enough, but _attribute_ has meaning in TF itself - ie: a _resource attribute_. Maybe something like "single custom attribute", etc.